niv-rozanowich/tvmaze/shows.js

```diff
@@ -0,0 +1,307 @@
+export const shows = [
```

Comment:
you could use the same shows file from the root directory.

Reply:
You can also remove this file..

Reply:
Im not sure how to remove files

Reply:
Remove the file, then type git status for instructions.